extensions/amp-social-share/0.1/amp-social-share.css

```diff
@@ -70,3 +70,7 @@ amp-social-share {
 .amp-social-share-whatsapp {
   background-color: #25d366;
 }
+
+.amp-social-share-system {
```

Comment:
I feel like I'm missing something obvious: Where does the share icon come from that is in your example screen?